File: 프로그래머스/lv0/120811. 중앙값 구하기/중앙값 구하기.py
def solution(array):
    median = 0
    idx = 0
    array = sorted(array)
    
    if len(array) % 2 == 0:
        idx = len(array) // 2
        median  = (array[idx - 1] + array[idx]) / 2
        
    else:
        idx = len(array) // 2
        median = array[idx]
        
    return median

File: 프로그래머스/lv0/120811. 중앙값 구하기/중앙값 구하기.swift
import Foundation

var median: Int = 0
var idx: Int = 0
func solution(_ array:[Int]) -> Int {
    let line = array.sorted()
    if (line.count) % 2 == 0 {
        idx = line.count / 2
        median = (line[idx - 1] + line[idx]) / 2
    } else {
        idx = line.count / 2
        median = line[idx]
    }
    return median
}
